class.jetpack.php

```diff
@@ -486,15 +486,9 @@ private function __construct() {
 		// If any module option is updated before Jump Start is dismissed, hide Jump Start.
 		add_action( 'update_option', array( $this, 'jumpstart_has_updated_module_option' ) );
 
-		// Identity Crisis AJAX callback function
-		add_action( 'wp_ajax_jetpack_resolve_identity_crisis', array( $this, 'resolve_identity_crisis_ajax_callback' ) );
-
 		// JITM AJAX callback function
 		add_action( 'wp_ajax_jitm_ajax',  array( $this, 'jetpack_jitm_ajax_callback' ) );
 
-		add_action( 'wp_ajax_jetpack_admin_ajax',          array( $this, 'jetpack_admin_ajax_callback' ) );
```

Comment:
@dereksmart there's a reference to this `jetpack_admin_ajax` here https://github.com/Automattic/jetpack/blob/a1fb89c9206fa12e6af8c8c43af1164d2086d8c7/_inc/jp.js#L251